clients/da-vinci-client/src/main/java/com/linkedin/davinci/notifier/LogNotifier.java

```diff
@@ -11,76 +12,76 @@ public class LogNotifier implements VeniceNotifier {
   }
 
   @Override
-  public void endOfPushReceived(String kafkaTopic, int partitionId, long offset, String message) {
-    logMessage("Received END_OF_PUSH", kafkaTopic, partitionId, offset, message, null);
+  public void endOfPushReceived(String pubSubTopic, int partitionId, long offset, String message) {
+    logMessage("Received END_OF_PUSH", pubSubTopic, partitionId, offset, message, null);
   }
 
   @Override
-  public void topicSwitchReceived(String kafkaTopic, int partitionId, long offset, String message) {
-    logMessage("Received TOPIC_SWITCH", kafkaTopic, partitionId, offset, message, null);
+  public void topicSwitchReceived(String pubSubTopic, int partitionId, long offset, String message) {
+    logMessage("Received TOPIC_SWITCH", pubSubTopic, partitionId, offset, message, null);
   }
 
   @Override
-  public void dataRecoveryCompleted(String kafkaTopic, int partitionId, long offset, String message) {
-    logMessage("Data recovery completed", kafkaTopic, partitionId, offset, message, null);
+  public void dataRecoveryCompleted(String pubSubTopic, int partitionId, long offset, String message) {
+    logMessage("Data recovery completed", pubSubTopic, partitionId, offset, message, null);
   }
 
   @Override
-  public void startOfIncrementalPushReceived(String kafkaTopic, int partitionId, long offset, String message) {
-    logMessage("Received START_OF_INCREMENTAL_PUSH", kafkaTopic, partitionId, offset, message, null);
+  public void startOfIncrementalPushReceived(String pubSubTopic, int partitionId, long offset, String message) {
+    logMessage("Received START_OF_INCREMENTAL_PUSH", pubSubTopic, partitionId, offset, message, null);
   }
 
   @Override
-  public void endOfIncrementalPushReceived(String kafkaTopic, int partitionId, long offset, String message) {
-    logMessage("Received END_OF_INCREMENTAL_PUSH", kafkaTopic, partitionId, offset, message, null);
+  public void endOfIncrementalPushReceived(String pubSubTopic, int partitionId, long offset, String message) {
+    logMessage("Received END_OF_INCREMENTAL_PUSH", pubSubTopic, partitionId, offset, message, null);
   }
 
   @Override
-  public void catchUpVersionTopicOffsetLag(String kafkaTopic, int partitionId) {
-    logMessage("Received CATCH_UP_BASE_TOPIC_OFFSET_LAG", kafkaTopic, partitionId, null, "", null);
+  public void catchUpVersionTopicOffsetLag(String pubSubTopic, int partitionId) {
+    logMessage("Received CATCH_UP_BASE_TOPIC_OFFSET_LAG", pubSubTopic, partitionId, null, "", null);
   }
 
   private void logMessage(
       String header,
-      String kafkaTopic,
+      String pubSubTopic,
       int partitionId,
       Long offset,
       String message,
       Exception ex) {
     if (ex == null) {
       LOGGER.info(
-          "{} for store {} user partitionId {}{}{}",
+          "{} for replica: {}{}{}",
           header,
-          kafkaTopic,
+          Utils.getReplicaId(pubSubTopic, partitionId),
+          pubSubTopic,
           partitionId,
```

Comment:
Note that here we _remove_ one of the `{}` but we _add_ an extra param to the logger function... the end result is some weird concatenation, with the last couple of params completely omitted. For example:

```
Received END_OF_PUSH for replica: HB_VPJtarget_prod-venice-3_v56784-15HB_VPJtarget_prod-venice-3_v5678415
```

I have not looked at the whole commit but this particular line jumped at me while reading logs.

Reply:
Thanks for reporting it. I've added fix for this in: https://github.com/linkedin/venice/pull/1113